camp/nfutsum/Twisty/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot;
+
+import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.wpilibj.GenericHID;
+import edu.wpi.first.wpilibj.Joystick;
+import edu.wpi.first.wpilibj.XboxController;
+import frc.robot.commands.MotorGo;
+import frc.robot.commands.StopTheMotor;
+import frc.robot.subsystems.NaherNeoMotor;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.CommandScheduler;
+import com.revrobotics.REVPhysicsSim;
+
+/**
+ * This class is where the bulk of the robot should be declared. Since Command-based is a
+ * "declarative" paradigm, very little robot logic should actually be handled in the {@link Robot}
+ * periodic methods (other than the scheduler calls). Instead, the structure of the robot (including
+ * subsystems, commands, and button mappings) should be declared here.
+ */
+public class RobotContainer {
+  // The robot's subsystems and commands are defined here...
+  private final NaherNeoMotor myMotor = new NaherNeoMotor();
+
+  private final StopTheMotor m_autoCommand = new StopTheMotor(myMotor);
+  private final Joystick myJoystick = new Joystick(0);
```

Comment:
The joystick ID should be stored in Constants.java

Reply:
+1